apps/rush-lib/src/cli/logic/PublishUtilities.ts

```diff
@@ -517,10 +535,26 @@ export default class PublishUtilities {
         for (const depName of downstreamNames) {
           const pkg: IPackageJson = allPackages.get(depName)!.packageJson;
 
-          PublishUtilities._updateDownstreamDependency(pkg.name, pkg.dependencies, change, allChanges, allPackages,
-            prereleaseToken, projectsToExclude);
-          PublishUtilities._updateDownstreamDependency(pkg.name, pkg.devDependencies, change, allChanges, allPackages,
-            prereleaseToken, projectsToExclude);
+          PublishUtilities._updateDownstreamDependency(
```

Comment:
>( [](start = 54, length = 1)

(out of scope for this PR)

It's error-prone to have functions with tons of parameters.  Eventually we should convert them into options interfaces.  And then you could use your spread operator trick here.

Reply:
Agreed. We should refactor a bunch of these.

---
In reply to: [178237385](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r178237385) [](ancestors = 178237385)